be/src/exec/schema_scanner/schema_load_tracking_logs_scanner.cpp

```diff
@@ -75,7 +76,7 @@ Status SchemaLoadTrackingLogsScanner::fill_chunk(ChunkPtr* chunk) {
         auto& info = _result.trackingLoads[_cur_idx];
         for (const auto& [slot_id, index] : slot_id_to_index_map) {
             if (slot_id < 1 || slot_id > 5) {
-                return Status::InternalError(fmt::format("invalid slot id:{}}", slot_id));
+                return Status::InternalError(strings::Substitute("invalid slot id: $0", slot_id));
```

Comment:
do you have any preference between fmt::format and this substitute?

Reply:
clang report `InternalError need const expr`, which should add `fmt:runtime`, but this is not support at current version of fmt. So here replace fmt with substitue.